### DISCUSSION TITLE
Fairy Types?

### DISCUSSION THREAD

**Discussion by matthewsam2** [Deijin27/RanseiLink#76] [Ideas]
Hello. I have some experience with ARM assembly. I am willing to help decode files if it will help add fairy type support to RanseiLink.

**Comment by Deijin27** [comment]
If you could figure out how to edit it that would be really cool.

I don't really know where to start, and I have more fun programming tools so I haven't really dug into it.
I do know MathyFurret has done a bit of reverse engineering of the game's code using Ghidra, which enabled him to figure out the damage formula https://docs.google.com/document/d/1C8IFx4ktKEyRJLU-5gt4IzdGntBVCY8_tlbojwfwj3A/edit

Another good place to start looking is other editors that already do some patches to their respective games such as SkyTemple
https://github.com/SkyTemple/skytemple-files/blob/master/skytemple_files/hardcoded/default_starters.py
